code/API_definitions/qod-api.yaml

```diff
@@ -730,83 +730,53 @@ components:
         - Gbps
         - Tbps
 
-    EventNotification:
-      type: object
+    CloudEvent:
+      description: Event compliant with the CloudEvents specification
       required:
-        - event
+        - id
+        - source
+        - specversion
+        - type
+        - time
       properties:
-        event:
-          $ref: "#/components/schemas/Event"
-        eventSubscriptionId:
+        id:
+          description: Identifier of this event, that must be unique in the source context.
           type: string
-          description: 'The ID of explicit subscription or none in case of implicit subscription'
-      example:
-        {
-          "event": {
-            "eventId": "5698d710-9b1b-4695-a958-7b228f08128c",
-            "eventType": "QOS_STATUS_CHANGED",
-            "eventTime": "2023-05-30T10:18:28Z",
-            "eventDetail": {
-              "sessionId": "7698d710-98b7-4695-a958-7b228f08128c",
-              "qosStatus": "UNAVAILABLE",
-              "statusInfo": "DURATION_EXPIRED"
-            }
-          }
-        }
-
-    Event:
-      description: The event being notified
-      type: object
-      required:
-        - eventType
-        - eventTime
-      properties:
-        eventId:
-          $ref: "#/components/schemas/EventId"
-        eventType:
-          $ref: "#/components/schemas/EventType"
-        eventTime:
-          $ref: "#/components/schemas/EventTime"
+        source:
+          description: Identifies the context in which an event happened in the specific Provider Implementation.
+          type: string
+          format: uri-reference
+        type:
+          description: The type of the event.
+          type: string
+          enum:
+            - 'org.camaraproject.sim-swap.v0.swapped'
```

Comment:
Must be org.camaraproject.sim-swap.v0.swapped.qod.v0.qosStatusChanged

Reply:
@bigludo7 I suppose you meant `org.camaraproject.qod.v0.qosStatusChanged`, right?

Reply:
@hdamker Yes ;)

Reply:
Thanks for the catch, but the correct value is "org.camaraproject.qod.v0.qos-status-changed", as we are using kebab-case for events